docs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "strong-xstats",
```

Comment:
Of the ones above, strong-express-metrics seems best to me. It reasonably scopes the metrics to express (there are many kinds of metrics).